sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/taskdata/EnvUtils.java

```diff
@@ -23,11 +23,11 @@ private EnvUtils() {
      * In the event of duplicate labels, the last duplicate wins.
      * This is the inverse of {@link #toProto(Map)}.
      */
-    public static Map<String, String> toMap(Environment environment) {
+    public static Map<String, Environment.Variable> toMap(Environment environment) {
```

Comment:
Maybe just fold this into withEnvVar? Looks like nothing else is calling it. The javadoc is also slightly inaccurate now as it's not quite the inverse of `toProto()` anymore.

Reply:
toMap does get called by a few tests that I changed to handle the new type passed in, so I was going to keep it. The tests need it anyway.